src/libraries/PositionConfig.sol

```diff
@@ -3,33 +3,9 @@ pragma solidity ^0.8.24;
 
 import {PoolKey} from "@uniswap/v4-core/src/types/PoolKey.sol";
 
-// A PositionConfig is the input for creating and modifying a Position in core, whose truncated hash is set per tokenId
+// A PositionConfig is the input for creating and modifying a Position in core
```

Comment:
Keeping this for now to reduce merge conflicts

later can move to test utils

Reply:
woudl rather delete now that we arent using it